### PR TITLE
feat: add new emulsify-core, and base theme docs

### PR DESCRIPTION
**Pull request by callinmullaney**
**This PR does the following:**
- Adds Emulsify Core documentation
- Reworks Emulsify Drupal documentation to reference the new 5.x base theme approach
- Adds various Emulsify starter documentation that is used by Emulsify CLI

### Functional Testing

- [ ] checkout pull request and run `npm run build && npm run start` - verify a localhost PR is provided
- [ ] Review the project documentation starting with http://localhost:3000/docs/emulsify-drupal

### CODE REVIEW

File: src/pages/docs/supporting-projects/emulsify-core/advanced-usage/3rd-party-libraries.md [hunk 1]
```diff
@@ -111,45 +115,23 @@ Add the new JS file to the `components/02-molecules/status/status.stories.js` an
...
-+   js:
-+     dist/js/02-molecules/status/status.js: {}
-+   dependencies:
-+     - core/drupal
-+     - core/jquery
-```
-
-Make sure to include the drupal and jquery core libraries as dependencies. They will both be loaded before your JS file in Drupal.
-
-The library then needs to be loaded in the component twig template. In this case edit `components/02-molecules/status/status.stories.twig` and add:
```
Comment (callinmullaney) [inline src/pages/docs/supporting-projects/emulsify-core/advanced-usage/3rd-party-libraries.md:132]: Adding this Drupal specific library attachment directly to a component twig file is not recommended. While this won't cause any errors it's mixing component files with CMS specific functions. Ideally, this is added to the template twig file this component is wired to.

File: src/pages/docs/supporting-projects/emulsify-core/advanced-usage/3rd-party-libraries.md [hunk 2]
```diff
@@ -4,7 +4,11 @@ pageTitle: 'Emulsify Drupal: Adding 3rd-party libraries'
 description: How to add libraries like jQuery to your project.
 ---
 
-## Global Libraries \(e.g. jQuery\)
```
Comment (callinmullaney) [inline src/pages/docs/supporting-projects/emulsify-core/advanced-usage/3rd-party-libraries.md:7]: We need to add this feature to Emulsify Core. Hiding these examples until this feature is publish in Emulsify Core 2.4.

File: src/pages/docs/emulsify-starter/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+title: Emulsify Starter
+pageTitle: Emulsify Starter
+description: No description. ''
+---
+
+Emulsify Starter is a scaffolding repository for the Emulsify CLI. It creates a platform-agnostic build that is meant for a standalone installation. It provides Emulsify Core integration for development, available configuration file overrides, and the various `npm run ...` commands to develop/test/build your project.
+
+## Installation
+
+`emulsify init [project_name] --platform none .`
```
Comment (amazingrando) [inline src/pages/docs/emulsify-starter/index.md:11]: When I run emulsify init [project_name] --platform none . (changing "[project_name]" to something else) as a standalone installation, I get the following error:

```emulsify init testname --platform none .
[==========                                                                                          ] 10% using starter for none, validating config
 ERROR  Unable to find an Emulsify starter for your project. Please specify one using the --starter flag: emulsify init myTheme --starter https://github.com/emulsify-ds/emulsify-drupal.git```

**Review by amazingrando — APPROVED** [review]
:shipit:

**Pull request merged**